apps/gallery/js/thumbnail_item.js

```diff
@@ -38,8 +38,7 @@ function ThumbnailItem(fileData) {
 ThumbnailItem.formatter = new navigator.mozL10n.DateTimeFormat();
 
 ThumbnailItem.prototype.localize = function() {
-  var date = new Date(this.data.date);
-  var imageCreatedAt = navigator.mozL10n.get('image-created-at');
-  var label = ThumbnailItem.formatter.localeFormat(date, imageCreatedAt);
+  var label = MediaUtils.shortDescription(
```

Comment:
Would this method be better named getAriaLabel()? Or is the point that it should be a useful description even outside of accessibility contexts?